jcl/src/java.base/share/classes/com/ibm/oti/util/Util.java

```diff
@@ -210,4 +213,114 @@ public static boolean doesClassLoaderDescendFrom(ClassLoader currentLoader, Clas
 	}
 	return true;
 }
+
+/**
+ * Provide a way of printing text without
+ * allocating memory, in particular without String concatenation.
+ * Used when printing a stack trace for an OutOfMemoryError.
+ * @param buf Buffer to receive string
+ * @param s New data for string
+ */
+public static void appendTo(Appendable buf, CharSequence s) {
+	try {
+		buf.append(s);
+	} catch (java.io.IOException e) {
+		/* ignore */
+	}
+}
+
+@SuppressWarnings("all")
+private static final String digits[]={"0", "1", "2", "3", "4", "5", "6", "7", "8", "9"}; //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$ //$NON-NLS-4$ //$NON-NLS-5$ //$NON-NLS-6$ //$NON-NLS-7$ //$NON-NLS-8$ //$NON-NLS-9$ //$NON-NLS-10$ 
+/**
+ * Helper method for output with PrintStream and PrintWriter
+ * @param buf Buffer to receive string
+ * @param number number to be converted to a sequence of chars
+ */
+public static void appendTo(Appendable buf, int number) {
+	int j = 1;
+	int i = 0;
+	for (i  = number; i >= 10; i /= 10) {
+		j *= 10;
+	}
+	Util.appendTo(buf, digits[i]);
+	while (j >= 10) {
+		number -= j * i;
+		j /= 10;
+		i = number / j;
+		Util.appendTo(buf, digits[i]);
+	}
+}
+
+/**
+ * Helper method to print text without constructing new String objects
+ * @param buf destination for new text
+ * @param condition value to print
+ */
+public static void appendTo(Appendable buf, boolean condition) {
+	Util.appendTo(buf, condition ? "true": "false"); //$NON-NLS-1$ //$NON-NLS-2$
+}
+
+/**
+ * Print a stack frame without allocating memory directly.
+ * @param e StackTraceElement to add
+ * @param elementSource location of the source file.  May be null.
+ * @param buf Receiver for the text
+ * @param extended add additional information
+ */
+public static void printStackTraceElement(StackTraceElement e, Object elementSource, Appendable buf, boolean extended) {
+	/*[IF Sidecar19-SE]*/
+	final java.lang.String modName = e.getMethodName();
```

Comment:
I expect this should be `e.getModuleName()`

Reply:
True.